### ISSUE TITLE
[Bug]: Visiting /find-placecal on a subdomain has bad styling

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/PlaceCal#2463]
## Description
e.g. https://exeter.placecal-staging.org/find-placecal

It's not explicitly linked but also easy to find. We should check routes on subdomains generally and lock them down. See also #2324 

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/861172/126aed08-9187-4448-b1cc-f74c297c8e13)

**Comment by kimadactyl** [comment]
I think in general there's a bigger ticket here around canonical URLs for things. like in this case it should prob be redirecting to the main domain and not be visible at all here. (all pages can be viewed on all partners if you type in the url directly)

**Comment by katjam** [comment]
> (all pages can be viewed on all partners if you type in the url directly)

Yes. This is the thing we should change. So things can't be "spoofed" as belonging in areas/ partnerships they don't belong in. It's a security issue that our API doesn't match the routing guards.

It's not a bit priority / high risk, but is sloppy.

**Comment by kimadactyl** [comment]
To be fixed as part of #3156 and #3163
